src/BenchmarksApps/Websocket/BenchmarkServer.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
```suggestion
    <TargetFramework>net6.0</TargetFramework>
    <TargetFramework Condition="'$(BenchmarksTargetFramework)' != ''">$(BenchmarksTargetFramework)</TargetFramework>
```

Reply:
No, I had it removed, it's not required since last year. Some of our projects might still have it, but the agent is able to inject it.